apps/devhub/templates/devhub/api/key.html

```diff
@@ -62,6 +62,7 @@
       </fieldset>
       <div class="listing-footer">
         <button id="generate-key" class="button prominent" type="submit">{{ _('Generate New Credentials') }}</button>
+        <button id="show-key" class="button prominent show-key" type="button">{{ _('Show Credentials') }}</button>
```

Comment:
instead of put the button here, why not replace the `jwtsecret` field with the button? That way the first page view would render a button 'show secret' and it would show it. Also, I don't see a reason why you need to show/hide the public ID. That is not something that needs to be protected. We only need to show/hide the secret.